src/applications/check-in/day-of/components/TravelPayReimbursementLink.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+function TravelPayReimbursementLink() {
+  return (
+    <div className="vads-u-margin-top--3">
+      <a
+        href="/health-care/get-reimbursed-for-travel-pay/"
```

Comment:
@mdewey Is this how I should be making this link? Or should it have the full path instead like: `https://www.va.gov/health-care/get-reimbursed-for-travel-pay/` ?

Reply:
Yeah that is fine